src/services/userSigninSignup.js

```diff
@@ -0,0 +1,96 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import uuid from 'uuid/v1';
+import dotenv from 'dotenv';
+import database from '../models/index';
+
+dotenv.config();
+
+const saltRounds = 10;
+const users = database.User;
+
+/**
+ * Checks if a user exist
+ * @param {Object} criteria - Criteria to validate user's existence
+ * @return {Boolean} - if user exist or not
+ */
+export const doesUserExist = (criteria) => {
+    const user = await users.findAll({
+    where: criteria
+  });
+  return !!user.length;
+}
+
+/**
+ * Signs in a user
+ * @param {Object} request - The request object to the server
+ * @param {Object} response - The response object from the server
+ * @param {Function} Next - Calls the express error handler
+ * @return {Void} - Returns nothing
+ */
+export const createUser = async (request, response, Next) => {
+  if (!doesUserExist({ email: request.body.email })) {
+    const hash = await bcrypt.hashSync(request.body.password, saltRounds);
+    const userData = { ...request.body, password: hash, id: uuid() };
```

Comment:
can we create a default value for id in the model?

Reply:
I dont think that is necessary